python/ray/serve/constants.py

```diff
@@ -34,3 +34,6 @@
     2000,
     5000,
 ]
+
+#: Name of backend reconfiguration method implemented by user.
+BACKEND_RECONFIGURE_METHOD = "reconfigure"
```

Comment:
shouldn't this be `user_reconfigure`?

Reply:
Sure, I forgot we had picked that name but that name makes sense

Reply:
Oh I don't think we settled on a name, that's just the method I saw in other places in this PR

Reply:
Ah I see.  Before I had `user_reconfigure` as our internal method and `reconfigure` as the name of the method the user provides in the class.  I think I'll just make them both `reconfigure` for consistency.